gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/TraversalStrategies.java

```diff
@@ -63,6 +63,8 @@
  */
 public interface TraversalStrategies extends Serializable, Cloneable {
 
+    static Set<Class> LOADED = new HashSet<>();
```

Comment:
Shouldn't we use a collection that supports concurrent modification? Different threads might be adding items to the collection.

Also, we could move the declaration to `GlobalCache`?.

Reply:
ha - i was so excited that this changed worked i didn't bother to think too hard about what i was doing. good suggestions in both cases - i've made those changes now. thanks